logger/src/lib.rs

```diff
@@ -18,6 +18,12 @@ use std::path::{Path, PathBuf};
 
 const LOG_FILE_SIZE: u64 = 1024 * 500;
 
+#[cfg(any(test, feature = "test"))]
```

Comment:
```suggestion
// Log file name used by our tests. It is different that the regular log files name so that we don't
// inadvertently delete log files we want to keep when running tests.
#[cfg(any(test, feature = "test"))]
```